src/python/grpcio_observability/grpc_observability/server_call_tracer.cc

```diff
@@ -260,10 +260,11 @@ void PythonOpenCensusServerCallTracer::RecordEnd(
 
 grpc_core::ServerCallTracer*
 PythonOpenCensusServerCallTracerFactory::CreateNewServerCallTracer(
-    grpc_core::Arena* arena) {
+    grpc_core::Arena* arena, const grpc_core::ChannelArgs& args) {
```

Comment:
Will the new xds channel arg be added for Python? Or is it already exist?

Reply:
Yeah, so what I'm thinking is that in the xDS server builder, we'll just add a channel arg that will be sent to the ServerCallTracer. When you start working on this piece, we can talk about it.